### DISCUSSION TITLE
Do I need to call store.Abort when calling store.Commit returns an error?

### DISCUSSION THREAD

**Discussion by Zacama** [open-policy-agent/community#437] [OPA and Rego] [answered]
Hello, I am new to using OPA. I have a problem integrating OPA with Go.
Do I need to call store.Abort when calling store.Commit returns an error?

**Comment by charlieegan3** [comment, marked as answer]
Hey, I don't believe so. You should just need to handle the error of the commit.

See the interface here for a few comments on each function: https://pkg.go.dev/github.com/open-policy-agent/opa/storage#Store

**Reply by Zacama** [reply]
Thank you very much. I think I understand it.
